nmtwizard/preprocess/prepoperator.py

```diff
@@ -79,7 +84,13 @@ def build_operator(operator_config, global_config, process_type, state, override
     _add_lang_info(operator_params, global_config, "source")
     _add_lang_info(operator_params, global_config, "target")
 
-    return operator_type, operator_cls(operator_params, process_type, state)
+    name = operator_params.pop("name", None)
+    if name is None:
+        if index is not None:
+            name = "%s_%d" % (operator_type, index)
+        else:
+            name = operator_type
```

Comment:
Looks like this never actually happens ?
The purpose of name is to be a unique identifier, shouldn't this be an error ?

Reply:
Right, let's make the `index` argument required.